server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -103,7 +106,7 @@
    * The logger.
    */
   public static final Logger LOGGER = LogManager.getLogger(LOGGER_NAME);
- 
+  
```

Comment:
I commented on this in Slack and have updated our Eclipse configuration to automatically remove whitespace.